Syntax error: syntax error when compiling csmith

**Env**:

```text
OS: Ubuntu 22.04
Arch: aarch64
Commit : 645b0f8
```

**Command:**

```shell
git clone https://github.com/csmith-project/csmith.git
cd csmith
sudo apt install g++ cmake m4
cmake -DCMAKE_INSTALL_PREFIX=$HOME/local/csmith
make && make install
```

**Symptom**:

```text
[ 87%] Building CXX object src/CMakeFiles/csmith.dir/StatementReturn.cpp.o
[ 88%] Building CXX object src/CMakeFiles/csmith.dir/StringUtils.cpp.o
/home/fanglinglong/csmith/src/StringUtils.cpp: In static member function ‘static void StringUtils::chop(std::string&)’:
/home/fanglinglong/csmith/src/StringUtils.cpp:234:20: error: ‘s’ was not declared in this scope
  234 |         size_t p = s.find_first_not_of( " \t");
      |                    ^
/home/fanglinglong/csmith/src/StringUtils.cpp:232:27: warning: unused parameter ‘str’ [-Wunused-parameter]
  232 | StringUtils::chop(string& str)
      |                   ~~~~~~~~^~~
make[2]: *** [src/CMakeFiles/csmith.dir/build.make:958: src/CMakeFiles/csmith.dir/StringUtils.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:234: src/CMakeFiles/csmith.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

**Problem analysis:**

In csmith/src/StringUtils.cpp , the parameter name of the StringUtils::chop function is str, but the name used in the function is s. As a result, a compilation error is reported.

**Suggestions**:
Change the parameter name of the StringUtils::chop function from str to s.

--- a/src/StringUtils.cpp
+++ b/src/StringUtils.cpp
@@ -231,13 +231,13 @@ StringUtils::longlong2str(INT64 i)
 void
 StringUtils::chop(string& str)
 {
-	size_t p = s.find_first_not_of( " \t");
-	s.erase( 0, p); 
+	size_t p = str.find_first_not_of( " \t");
+	str.erase( 0, p); 
  
-	p = s.find_last_not_of( " \t");
+	p = str.find_last_not_of( " \t");
 	if (string::npos != p)
 	{
-		s.erase(p + 1);
+		str.erase(p + 1);
 	}
 }
 

